src/Layout.ts

```diff
@@ -1,61 +1,53 @@
-import { Graphics } from '@pixi/graphics';
 import { Container } from '@pixi/display';
 import { Content, LayoutOptions, Styles } from './utils/types';
 import { AlignController } from './controllers/align/AlignController';
 import { StyleController } from './controllers/StyleController';
 import { SizeController } from './controllers/SizeController';
 import { ContentController } from './controllers/ContentController';
-import { getColor } from './utils/helpers';
-import { Sprite } from '@pixi/sprite';
 import { TextStyle } from '@pixi/text';
 
 /**
- * Universal layout class for Pixi.js.
+ * Layout controller class for any PixiJS Container based instance.
  *
- * You can consider layout as div from CSS.
+ * To be be used for automatic align and resize children tree,
+ * where every child behavior can be configured using css like configurations.
  *
- * It will be rendered as PIXI.Container and can be used for automatic align and resize blocks and text.
+ * Also it adds a list of css-like properties for styling like background style or text style,
+ * check {@link SizeController} class.
  *
- * Also it brings a list of css-like properties for styling itself and it's children.
+ * Any PixiJS Container based instance can be turned into a layout by calling {@link Layout#initLayout} method.
+ * @example
+ * const container = new Container().initLayout();
  *
- * Children will be resized and aligned to fit parent size, if they have width and height properties
+ * container.layout?.setStyles({
+ *      background: 'black',
+ *      width: '100%',
+ *      height: '100%',
+ *      padding: 10,
+ *      overflow: 'hidden',
+ *      color: 'white',
+ * }); // set styles
  *
- * (like Sprite or Graphics instances from Pixi.js)
- * @example
- * const layout = new Layout({
- * 	id: 'myLayout',
- * 	styles: {
- * 		width: 100,
- * 		height: 100,
- * 		background: 'red',
- * 	},
- * 	content: [
- * 		'Hello world',
- * 		{
- * 			id: 'innerLayout1',
- * 			text: 'Inner layout 1',
- * 		},
- * 		{
- * 			id: 'innerLayout2',
- * 			text: 'Inner layout 2',
- * 		},
- * 	],
- * 	globalStyles: {
- * 		innerLayout1: {
- * 			width: 200,
- * 			height: 200,
- * 		},
- * 		innerLayout1: {
- * 			width: 200,
- * 			height: 200,
- * 		},
- * 	},
- * });
+ * container.layout?.setContent({
+ *      text: 'Hello World',
+ * }); // set content
+ *
+ * Or alternatively:
+ *
+ * const layoutSystem = new LayoutSystem({
+ *      id: 'root',
+ *      content: 'Hello World',
+ *      styles: {
+ *          background: 'black',
+ *      }
+ * }); // create layout system
+ *
+ * app.stage.addChild(layoutSystem.container); // add layout system generated container to the stage
  */
-export class Layout extends Container
+export class LayoutSystem
```

Comment:
Renamed and now gets container as contractor param instead of extending it.

If the container is not provided, the system creates it, which can be accessed through the `.container` property.